src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let i = 0;
```

Comment:
```suggestion
    let index = 0;
```
Краще давати більш зрозумілі назви